docs/root/intro/arch_overview/http_routing.rst

```diff
@@ -24,6 +24,8 @@ request. The router filter supports the following features:
   level.
 * :ref:`Path <config_http_conn_man_route_table_route_path_redirect>`/:ref:`host
   <config_http_conn_man_route_table_route_host_redirect>` redirection at the route level.
+* :ref:`Direct HTTP responses <envoy_api_field_route.Route.direct_response_action>`,
```

Comment:
nit: incomplete sentence.

Reply:
That's intentional, to match the style of the rest of the items in the list.

Reply:
It reads a little weird to me TBH, but not a big deal.

Reply:
I rephrased this item in the latest commit to this PR. Does the new wording work better?